NRzure/dashboards/example-dashboard/Intern Hackathon Azure.json

```diff
@@ -0,0 +1,373 @@
+{
+    "name": "Intern Hackathon Azure",
```

Comment:
Let's get this a more descriptive name, something that tells us what kind of information it displays.